### ISSUE TITLE
[desktopintegration] Fails when AppImage has "(" or ")" in name

### ISSUE THREAD

**Issue by probonopd** [AppImage/AppImageKit#74]
Probably the value after `Exec=` in the generated `.desktop` file needs to be quoted to prevent from this.

**Comment by Sod-Almighty** [comment]
I'll take a look

**Comment by TheAssassin** [comment]
@Sod-Almighty any progress on this?

**Comment by Sod-Almighty** [comment]
No. I decided I didn't like being insulted and condescended to, so I spent my time doing other things.

**Comment by azubieta** [comment]
I guess it's please move it to the libappimage repo. And let's take another look to it there.
